src/tools/r2rdump/x86/UnwindInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text;
+using System.Xml.Serialization;
+
+namespace R2RDump.x86
+{
+    public class UnwindInfo : BaseUnwindInfo
+    {
+        public uint FunctionLength { get; set; }
+
+        public UnwindInfo() { }
+
+        public UnwindInfo(byte[] image, int offset)
```

Comment:
Is this all the information we save in the x86 R2R image? I'm surprised the x86 version is so short compared to the amd64 version.

Reply:
The UNWIND_INFO struct in win64unwind.h only contains ULONG FunctionLength for x86